django/contrib/contenttypes/forms.py

```diff
@@ -32,7 +29,7 @@ def __init__(self, data=None, files=None, instance=None, save_as_new=None,
     @classmethod
     def get_default_prefix(cls):
         opts = cls.model._meta
-        return '-'.join((opts.app_label, opts.model_name, cls.ct_field.name, cls.ct_fk_field.name))
+        return opts.app_label + '-' + opts.model_name + '-' + cls.ct_field.name + '-' + cls.ct_fk_field.name
```

Comment:
I think splitting this to more lines would increase readability. Probably the same for the above with `self.relname`.